### ISSUE TITLE
Show plugin location button vertical alignment

### ISSUE THREAD

**Issue by mbilker** [nylas/N1#1743]
- What operating system are you using?
**Arch Linux**

- What version of N1 are you using?
**0.4.14-master**

_Description:_
The plugins view in Preferences show "In Development" plugins and the Enable/Disable button was changed to a slider. Now, the show button is out of alignment with the slider.

![2016-03-15-11-42-37_611x88](https://cloud.githubusercontent.com/assets/1266541/13783886/890b5082-eaa3-11e5-97b9-8537cab04466.png)

**Comment by bengotow** [comment]
Hey! Is there any chance your plugin icon is too large and isn't being constrained to the same size as the others?

**Comment by mbilker** [comment]
The plugin icon is the same size as the other icons.

```
send-later/icon.png: PNG image data, 54 x 54, 8-bit/color RGBA, non-interlaced
/home/mbilker/.nylas/dev/packages/cypher/assets/icon.png: PNG image data, 54 x 54, 8-bit/color RGBA, non-interlaced
```

**Comment by mbilker** [comment]
From further analysis, it looks like the Show button is vertically centered, but the Enable/Disable slider is slightly above the center.

**Comment by bengotow** [comment]
Hey! This shipped in `0.4.19` a few minutes ago. You can grab the update by choosing "Check for Updates" from the menu or download the new version directly from the releases section of the project. Enjoy!
